html/semantics/scripting-1/the-script-element/module/referrer-strict-policies.sub.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Referrer with the strict-origin referrer policy</title>
+<meta name="referrer" content="strict-origin">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script type="module">
+
+// "name" parameter is necessary for bypassing the module map.
```

Comment:
```suggestion
// "name" parameter is necessary for bypassing the module map in the descendant import.
```

or something like that; I was confused since the two imports in this file are different URLs so don't seem to need a disambiguator.

Reply:
SGTM, yeah I just copied this from other similar tests